scripts/teardown-tests.sh

```diff
@@ -10,3 +10,4 @@ git submodule update --init
 echo -e "\nremoving compiled contracts/programs...\n"
 rm starkware
 rm -rf build/!(os_latest.json)
```

Comment:
This doesn't exist anymore right?

Reply:
Yes, I had the same thought but I hesitate to remove it, mostly because the whole thing is still a bit sloppy and if `os_latest.json` is left around it will consume a lot of disk space.

I could easily be persuaded to improve this if you have suggestions :)

Reply:
Ok, it was good to review `extglob` ("extended globbing") and the `!` operator, because this isn't what we want.

The `!` operator inverts the match, so this is effectively removing:

* everything but `os_latest.json`
* everything but `os_latest.json.gz`

Which is the same as removing everything. What we want is only the latter.

Reply:
Fixed in c1bb7be